backend/oasst_backend/prompt_repository.py

```diff
@@ -291,6 +292,26 @@ def store_ranking(self, ranking: protocol_schema.MessageRanking) -> Tuple[Messag
 
         return reaction, task
 
+    def insert_toxicity(self, message_id: UUID, model: str, toxicity) -> MessageToxicity:
```

Comment:
please provide a type annotation for the toxicity parameter, e.g. `dict[str, Any]`

Reply:
Since the function seems to expect a single float in `toxicity["label"]` .. it would be better to extract that value outside fo the function and pass simply a float (then it would also match the docstring again ;-) ).